ARM MariaDB permission error

**Bug description**
I followed the ARM guide on starting tutor with MariaDB.
Apparently there's a permission problem:

```
2022-02-18 05:50:33+00:00 [Note] [Entrypoint]: Initializing database files
2022-02-18  5:50:33 0 [Warning] Can't create test file /var/lib/mysql/ffffffff.lower-test
2022-02-18  5:50:33 0 [ERROR] mysqld: Can't create/write to file '/var/lib/mysql/aria_log_control' (Errcode: 13 "Permission denied")
2022-02-18  5:50:33 0 [ERROR] mysqld: Got error 'Can't create file' when trying to use aria control file '/var/lib/mysql/aria_log_control'
```

**How to reproduce**
Just follow the tutor for ARM guide.

**Environment**
Ubuntu 20.04 on ARM64 AWS instance

**Additional context**
I managed to solve it, the owner of `$HOME/local/share/tutor/data` was root:root (0:0) while the app inside the container is running as 999:999 so I chowned the data dir and it started correctly.

--- a/docs/tutorials/arm64.rst
+++ b/docs/tutorials/arm64.rst
@@ -22,9 +22,9 @@ Next, configure Tutor::
 
 Go through the configuration process, answering each question.
 
-Then, build the Open edX image::
+Then, build the "openedx" and "permissions" images::
 
-    tutor images build openedx
+    tutor images build openedx permissions
 
 If you want to use Tutor as an Open edX development environment, you should also build the development images::
 
@@ -45,18 +45,16 @@ Configure Tutor to use MariaDB::
 Finish setup and start Tutor
 ----------------------------
 
-From this point on, use Tutor as normal.
+From this point on, use Tutor as normal. For example, start Open edX and run migrations with::
 
-For example, to apply migrations and start Open edX::
-
+    tutor local start -d
     tutor local init
-    tutor local start
 
 Or for a development environment::
 
-    tutor local init
     tutor local stop
-    tutor dev start
+    tutor dev start -d
+    tutor dev init
 
 Using with tutor-mfe
 --------------------

